2_run/src/run_glm3.R

```diff
@@ -181,6 +201,7 @@ run_glm3_model <- function(sim_dir, nml_objs, model_config, burn_in, burn_out, e
         burn_out = burn_out,
         export_fl = NA,
         export_fl_hash = NA,
+        extraction_error = ifelse(exists("extraction_error"), extraction_error, NA),
```

Comment:
In what scenario would this object _not_ be created?

Reply:
I was just thinking through this myself! I think if the model errors out in the initial [`retry()` attempts](https://github.com/hcorson-dosch/lake-temperature-process-models/blob/add_extraction_error/2_run/src/run_glm3.R#L128-L142) to `run_glm()`, or if it results in a `glm_code`!=0 or `max_ouput_date`!=`param_sim_stop` and errors [here](https://github.com/hcorson-dosch/lake-temperature-process-models/blob/add_extraction_error/2_run/src/run_glm3.R#L146) then the `extraction_error` variable won't exist.

Reply:
```suggestion
        # If the error happened prior to the attempt at extraction, need to provide an NA here
        extraction_error = ifelse(exists("extraction_error"), extraction_error, NA),
```